Update spec for validators (#61)
Kattis/problem-package-format: spec/2023-07-draft/problem_package_format.md

| grading                                     | String                                                | default      | One of "default" and "custom".                                                                                                                                                                                                                                                                                    |
| grader\_flags                               | String                                                | empty string | arguments passed to the grader for this test data group.                                                                                                                                                                                                                                                          |

> | input\_validator\_flags |  String or map of strings to strings |  empty string  | arguments passed to each input validator for this test data group. If a string then those are the flags that will be passed to each input validator for this test data group. If a map then each key is the name of the input validator and the value is the flags to pass to that input validator for this test data group, which means the default flags will be passed to input validator that is not listed in the map.                       |

Reviewer: ```suggestion
| input\_validator\_flags |  String or map of strings to strings |  empty string  | arguments passed to each input validator for this test data group. If a string then those are the flags that will be passed to each input validator for this test data group. If a map then each key is the name of the input validator and the value is the flags to pass to that input validator for this test data group. Validators not present in the map are run without flags.                       |
```
